src/lib/github.ts

```diff
@@ -6,11 +6,12 @@ export class Github {
     // private static headers_githubRequest = [
     //     ["Accept", "application/vnd.github+json"],
     // ];
-    private static headers_githubRequestAuth = new Headers([
+
+    private static headers_githubRequestAuth = [
         ["Accept", "application/vnd.github+json"],
         ["Authorization", `token ${process.env.APIKEY_GITHUB}`]
-    ]);
-
+    ];
```

Comment:
I don't know if this was intentional/accidental due to my last commit but I would like to maintain the `new Headers([` here so it's in one place.